### PR TITLE
[PLAYER-3574] Added accessibility support to multiaudio and scrubbing.

### PR DESCRIPTION
**Pull request by IvanSakharovskii**
PLAYER-3368, PLAYER-3574. Added labels, that will be announced when someone presses multiaudio cell and CC cell. Also changed logic for announcing while scrubbing. 
No unit tests added, changed UI params

### CODE REVIEW

File: sdk/react/panels/ItemSelectionScrollView.js [hunk 1]
```diff
@@ -42,9 +44,12 @@ class ItemSelectionScrollView extends React.Component {
     const buttonStyle = isSelectedItem ? styles.selectedButton : styles.button;
     const textStyle = isSelectedItem ? styles.selectedButtonText : styles.buttonText;
     const checkmarkIcon = isSelectedItem ? this.props.config.icons.selected.fontString : "";
+    let accessibilityString = AccessibilityUtils.createAccessibilityLabelForCell(this.props.cellType, item);
```
Comment (MarkEPAM) [inline sdk/react/panels/ItemSelectionScrollView.js]: const

File: sdk/react/widgets/controlBarWidgets.js [hunk 2]
```diff
@@ -73,12 +71,14 @@ class controlBarWidget extends React.Component {
     var iconConfig = (options.volume > 0) ? options.iconOn : options.iconOff;
```
Comment (MarkEPAM) [inline sdk/react/widgets/controlBarWidgets.js]: let/const

File: sdk/react/widgets/controlBarWidgets.js [hunk 3]
```diff
@@ -73,12 +71,14 @@ class controlBarWidget extends React.Component {
...
     var fontFamilyStyle = {fontFamily: iconConfig.fontFamilyName};
```
Comment (MarkEPAM) [inline sdk/react/widgets/controlBarWidgets.js]: let/const

File: sdk/react/widgets/controlBarWidgets.js [hunk 4]
```diff
@@ -93,14 +93,14 @@ class controlBarWidget extends React.Component {
           <Text style={options.style}>{options.durationString}</Text>
         </TouchableHighlight>);
     } else {
-      var playHead = <Text style={options.playHeadTimeStyle} accessible={true} accessibilityLabel={options.playHeadTimeString + STRING_CONSTANTS.SECONDS}>{options.playHeadTimeString}</Text>;
-      var duration = <Text style={options.durationStyle} accessible={true} accessibilityLabel={options.durationString + STRING_CONSTANTS.TOTAL_SECONDS}>{options.durationString}</Text>;
+      const playHead = <Text style={options.playHeadTimeStyle} accessible={true} accessibilityLabel={options.playHeadTimeString + STRING_CONSTANTS.SECONDS}>{options.playHeadTimeString}</Text>;
+      const duration = <Text style={options.durationStyle} accessible={true} accessibilityLabel={options.durationString + STRING_CONSTANTS.TOTAL_SECONDS}>{options.durationString}</Text>;
       return (
         <View style={options.completeTimeStyle}>
         {playHead}
```
Comment (MarkEPAM) [inline sdk/react/widgets/controlBarWidgets.js]: indent

File: sdk/react/widgets/controlBarWidgets.js [hunk 5]
```diff
@@ -135,9 +135,9 @@ class controlBarWidget extends React.Component {
   moreOptionsWidget = (options) => {
     var fontFamilyStyle = {fontFamily: options.icon.fontFamilyName};
```
Comment (MarkEPAM) [inline sdk/react/widgets/controlBarWidgets.js]: const

File: sdk/react/bottomOverlay.js [hunk 6]
```diff
@@ -38,6 +40,8 @@ var progressBarHeight = 3;
 var scrubberSize = 14;
```
Comment (MarkEPAM) [inline sdk/react/bottomOverlay.js]: const's

**Review by MarkEPAM — APPROVED** [review]

**Review by akshaydch11 — APPROVED** [review]

File: sdk/react/accessibilityUtils.js [hunk 7]
```diff
@@ -0,0 +1,43 @@
+/**
+ * Sample React Native App
```
Comment (brol1dev) [inline sdk/react/accessibilityUtils.js:2]: You can remove these comments :)

**Review by brol1dev — APPROVED** [review]
Just added one inoffensive comment.

**Comment by brol1dev** [comment]
Merging, just look at the comment, it's trivial.

**Pull request merged**